docs/quick_tutorial/authentication.rst

```diff
@@ -111,14 +110,12 @@ are you) and authorization (what are you allowed to do) are not just pluggable,
 but decoupled. To learn one step at a time, we provide a system that identifies
```

Comment:
Aren’t they coupled in SecurityPolicy now?

Reply:
clarified in ef5b40196